content/learn/book/control-flow/change-detection.md

```diff
@@ -5,3 +5,170 @@ insert_anchor_links = "right"
 weight = 8
 status = 'hidden'
 +++
+
+Change detection is an inherent feature of Bevy's Entity Component System (ECS). Each time a
+[`Component`] or [`Resource`] is modified, Bevy marks the item as changed. This mechanism helps
+optimize performance by avoiding unnecessary system executions when no relevant changes have
```

Comment:
I don't think it avoids system executions, it just filters out items that haven't changed while you're iterating over the query

Reply:
Changed "system executions" to "calculations".